versioned_docs/version-23.10/installation/prerequisites.md

```diff
@@ -382,4 +382,4 @@ For very large amounts of data, contact your sales representative.
 
 ## Network flows
 
-If you have firewalls or security equipments in place, check the [table of network flows](./technical.md#tables-of-network-flows).
+If you have firewalls or security devices in place, check the [table of network flows](./technical.md#tables-of-network-flows).
```

Comment:
so, as device suggests a physical device, what other word could we use here?

Reply:
"resources"